packages/api-client/src/components/ImportCollection/hooks/useUrlPrefetcher.ts

```diff
@@ -12,34 +18,96 @@ export type PrefetchResult = {
  * Core logic for fetching and processing a URL
  */
 export function createUrlPrefetcher() {
-  async function prefetchUrl(value: string | null, proxy?: string) {
-    if (!value || typeof value !== 'string') {
-      return { state: 'idle', content: null, url: null, error: null }
+  async function prefetchUrl(input: string | null, proxy?: string) {
+    if (!input || typeof input !== 'string') {
```

Comment:
```suggestion
    if (!input || typeof input !== 'string') {
```
redundant check since !input should catch null as well

Reply:
redundant suggestion as I forgot to change it